src/sentry/integrations/slack/message_builder/base/base.py

```diff
@@ -36,14 +36,17 @@ def build(self) -> SlackBody:
         """Abstract `build` method that all inheritors must implement."""
         raise NotImplementedError
 
-    def build_fallback_text(self, obj: Group | Event, project_slug: str) -> str:
+    def build_fallback_text(self, obj: Group | Event | GroupEvent, project_slug: str) -> str:
```

Comment:
Actually not sure if `Event` is used outside of tests, but I was passing a `GroupEvent` so I added this.